src/main/java/com/gigacal/entity/EventEntity.java

```diff
@@ -0,0 +1,45 @@
+package com.gigacal.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Table(name = "EVENTS")
```

Comment:
Have we decided on uppercase table names? In @cristian0x pr names are lowercase - besides, this pull request can be done in a one with first merge request

Reply:
For now, I have left uppercase names. Changes merged to master.